hypha/apply/projects/filters.py

```diff
@@ -43,9 +43,9 @@ class ProjectListFilter(filters.FilterSet):
         (1, 'Behind schedule'),
     )
 
-    fund = Select2ModelMultipleChoiceFilter(label='Funds', queryset=get_used_funds)
-    lead = Select2ModelMultipleChoiceFilter(label='Lead', queryset=get_project_leads)
-    status = Select2MultipleChoiceFilter(label='Status', choices=PROJECT_STATUS_CHOICES)
+    project_fund = Select2ModelMultipleChoiceFilter(label='Funds', queryset=get_used_funds)
+    project_lead = Select2ModelMultipleChoiceFilter(label='Lead', queryset=get_project_leads)
+    project_status = Select2MultipleChoiceFilter(label='Status', choices=PROJECT_STATUS_CHOICES)
```

Comment:
@sandeepsajan0, this fixes the ui but breaks the filter functionality, as these fields are not model fields anymore. We may need to use [to_field_name](https://django-filter.readthedocs.io/en/stable/ref/filters.html#to-field-name) param here.